CPAC/pipeline/cpac_pipeline.py

```diff
@@ -1006,7 +1006,7 @@ def build_workflow(subject_id, sub_dict, c, pipeline_name=None, num_ants_cores=1
                 raise Exception
 
             flirt_reg_anat_mni = create_fsl_flirt_linear_reg(
-                'anat_mni_flirt_register_%d' % num_strat
+                'anat_mni_flirt_register_{0}'.format(num_strat)
```

Comment:
Since we're in Python 3 now, this could be the even simpler
```Python
f'anat_mni_flirt_register_{num_strat}'
```

Reply:
Done! Thanks for suggestion.